testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js

```diff
@@ -178,6 +178,9 @@ module("value", moduleConfig, () => {
     test("should not throw an error when the value is \"null\" or \"undefine\"", assert => {
         const errorLogStub = sinon.stub(errors, "log");
 
+        errorLogStub.withArgs("W1002").returns(true)
+            .throws("Non W1002 Exception");
```

Comment:
There are no exceptions throws in this test case.
The behavior has been changed and now the warning throws in debug scripts only. It is not a breaking change